static/js/public.js

```diff
@@ -121,3 +121,37 @@ jQuery(document).ready(function ($) {
     emailBody: description + CURRENT_PAGE_URL
   });
 });
+
+/**
+ * Set url hash if hash provided in the url,
+ * or set hash based on the active panel
+ */
+$(function(){
+  $('.mdl-tabs__tab').click(function(){
```

Comment:
Right now it doesn't scroll up to the top. Does it make sense to add it  anyway?